src/protagonist/Engine/Ingest/AssetIngester.cs

```diff
@@ -64,7 +64,7 @@ public Task<IngestResult> Ingest(LegacyIngestEvent request, CancellationToken ca
         catch (Exception e)
         {
             logger.LogError(e, "Exception ingesting IncomingIngest - {Message}", request.Message);
-            return Task.FromResult(new IngestResult(internalIngestRequest?.Asset, IngestResultStatus.Failed));
+            return Task.FromResult(new IngestResult(new Asset() { Id = internalIngestRequest?.Id}, IngestResultStatus.Failed));

```

Comment:
Does this not break `IngestHandler.UpdateCustomerQueue()` as it needs `Customer` value?

Would it be possible to change `IngestResult` to just store the AssetId? As you can get `customer` from that and it's obvious that `IngestResult` is only ever the Id?